spec/jobs/discovery_report_job_spec.rb

```diff
@@ -2,26 +2,29 @@
 
 RSpec.describe DiscoveryReportJob, type: :job do
   let(:job) { described_class.new }
-  let(:job_run) { create(:job_run) }
+  let(:job_run) { create(:job_run, :discovery_report) }
   let(:outfile) { 'tmp/foo.out' }
 
   before { allow(job_run.to_discovery_report).to receive(:output_path).and_return(outfile) }
 
   after { FileUtils.rm(outfile) if File.exist?(outfile) } # cleanup
 
   describe '#perform' do
-    let(:jbuilder) { instance_double(Jbuilder, target!: '{"x":1}') } # mock the expensive stuff
+    context 'when success' do
```

Comment:
I think we also want a test for error case, too, no?  one for status completed and one with status error?  Or is this the wrong place to test that?  

It would be awesome to have a test for "waiting" status as well -- you'd have to do some serious interception of methods to get it to stay in that state, tho, I guess.

Reply:
this context block was a holdover from when I had a rescue block in the discovery report too in this PR, we will want a success/failure context later when it gets re-added, but not needed now, so will remove